core/config/engine.h

```diff
@@ -59,13 +59,17 @@ class Engine {
 	double _process_step = 0;
 
 	int ips = 60;
+	int ips_override = 60;
 	double physics_jitter_fix = 0.5;
 	double _fps = 1;
 	int _max_fps = 0;
 	int _audio_output_latency = 0;
+	double _final_time_scale = 1.0;
 	double _time_scale = 1.0;
+	double _time_scale_override = 1.0;
```

Comment:
Why have you reverted this change? The names `_game_time_scale`, `_user_time_scale` and `_effective_time_scale` convey the meaning better, imo. I have given my reasoning above.

Reply:
The priority for names should be that they succinctly convey the meaning, without risk of misinterpretation.
`_time_scale_override` could be misinterpreted for replacing the current timescale instead of modulating it.
`_time_scale` could be misinterpreted for being the actual running time scale, instead of just the 'game' provided component of it.
`_time_scale_final` could be misinterpreted for being an unchanging variable, since `final` is commonly associated with something that doesn't change.

The same misinterpretations do not hold for `_game_time_scale`, `_user_time_scale` and `_effective_time_scale`.